src/Service/VrijBrpService.php

```diff
@@ -157,15 +157,17 @@ public function createCaseNotification(array $data, array $config): array
             $array = $object->toArray(['embedded' => true]);
 
             if (in_array(needle: $array['toelichting'], haystack: ['intra_mun_relocation', 'inter_mun_relocation']) === true) {
-                $this->extendSync($object, $array);
+                $object = $this->extendSync($object, $array);
             }
 
+            $array = $object->toArray(['embedded' => true]);
+
             $now           = new DateTime();
             $message       = [
-                'kanaal'       => 'zaak.created',
+                'kanaal'       => 'zaken',
                 'hoofdObject'  => $object->getUri(),
-                'resource'     => 'Zaak',
-                'resourceUrl'  => $object->getUri(),
+                'resource'     => 'rol',
+                'resourceUrl'  => $array['rollen'][0]['url'],
```

Comment:
no dot array 🤔 rollen.0.url ?

Reply:
It's we're in an toArray version of the object here